src/event_impl.cc

```diff
@@ -61,7 +61,7 @@ NAN_METHOD(EventImpl::SetupEvent)
     AddTimeout, UpdateTimeout, RemoveTimeout
   );
 
-  NanReturnUndefined();
+  return;
```

Comment:
`return Nan::Undefined();` ? I'll stop here and wait for your response since I haven't touched nan2 yet :)